apps/frontend-manage/src/components/common/ColorPicker.tsx

```diff
@@ -0,0 +1,69 @@
+import { Button, Label, ThemeContext } from '@uzh-bf/design-system'
+import { useContext, useState } from 'react'
+import { HexColorInput, HexColorPicker } from 'react-colorful'
+import { PRESET_COURSE_COLORS } from 'src/constants'
+
+interface Props {
+  color: string
+  onSubmit: (newColor: string) => void
+  onAbort: () => void
+}
+
+function ColorPicker({ color, onAbort, onSubmit }: Props) {
+  const theme = useContext(ThemeContext)
+  const [newColor, setNewColor] = useState(color)
+
+  return (
+    <div className="absolute flex flex-col p-1 bg-white rounded-md shadow-md w-80 left-10 top-8">
+      <HexColorPicker
+        style={{ width: 'auto' }}
+        color={newColor}
+        onChange={setNewColor}
+      />
+      <div className="grid grid-cols-5 gap-0.5 pt-3 pb-3 justify-items-center">
+        {PRESET_COURSE_COLORS.map((presetColor, index) => (
+          <button
```

Comment:
Is there a specific reason not to use our custom <Button> component here (just for consistency)?

Reply:
Nope, didn't notice it but thank you for the adjustment 😄

Reply:
Thanks 👍